lzma-static.cabal

```diff
@@ -65,7 +65,7 @@ library
   other-modules:       LibLzma
 
   build-depends:       base       >=4.5    && <5
-                     , bytestring >=0.9.2  && <0.11
+                     , bytestring >=0.11.1
```

Comment:
Why `>=`?

Reply:
base 4.16 for ghc 9.2.1

Reply:
https://hackage.haskell.org/package/bytestring-0.11.1.0

as you can see, this is the first version that allows base > 4.15

Reply:
No, you're disallowing builds with older bytestrings. Why?

Reply:
0.11.0 is still a lower base bound.

Reply:
> 0.11.0 is still a lower base bound.

I don't understand. Why only use `>= 0.11.1` if this packages fails to build with `< 0.11.1`. But that's not the case.

Reply:
i see, we still want those.

so `< 0.12` should be the right thing here i guess